rebar.config.script

```diff
@@ -1,9 +1,9 @@
 IsRebar3 = erlang:function_exported(rebar3, main, 1),
 
 Rebar2Deps = [
-    {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.8.0"}}},
-    {jsone, ".*", {git, "https://github.com/sile/jsone.git", {tag, "1.5.2"}}},
-    {esq, ".*", {git, "https://github.com/fogfish/esq.git", {tag, "2.0.4"}}}
+    {lager, "3.9.2"},
```

Comment:
rebar2 supports this format for dependencies ? ({lager, "3.9.2"},)

Reply:
Hi! I just tried to change sources to hex, and it works, but Im not sure if it's the recommended way (I'm Elixir developer), please tell me if I'm doing it wrong.

Actually I have another problem and maybe we can fix it here:
Using Hex.pm it doesn't pull rebar.config.script, and then when running in elixir it prints a warning.